<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntryFilterSettings</class>
 <widget class="QWidget" name="EntryFilterSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>44</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="enableFilterCheckBox">
       <property name="text">
        <string>Enable filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="maxDepthSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="maxDepth" stdset="0">
        <UInt>10</UInt>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="maxDepthSpinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="enableCriticalPathFilterCheckBox">
       <property name="text">
        <string>Critical path filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="criticalPathThresholdHorizontalSlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="criticalPathThresoldSpinBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
